mozjs/src/rust.rs

```diff
@@ -400,16 +400,15 @@ impl Runtime {
         glob: HandleObject,
         script: &str,
         filename: &str,
-        line_num: u32,
         rval: MutableHandleValue,
+        options: CompileOptionsWrapper,
     ) -> Result<(), ()> {
         debug!(
             "Evaluating script from {} with content {}",
             filename, script
         );
```

Comment:
the only reason why we pass filename here is this debug statement, can we just extract filename from `CompileOptionsWrapper`?

Reply:
I think just doing c_string.to_string_lossy() would be enough.